<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teostratimerwindow</class>
 <widget class="QMainWindow" name="teostratimerwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Timer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>9</y>
      <width>140</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>140</width>
      <height>50</height>
     </size>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Warn me at:</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="warning_input">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>49</width>
       <height>21</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>51</height>
      </size>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>warning_input</zorder>
    <zorder>groupBox_2</zorder>
   </widget>
   <widget class="QPushButton" name="start_timer">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>85</width>
      <height>42</height>
     </rect>
    </property>
    <property name="text">
     <string>Start
Timer</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <widget class="QLabel" name="time_left_text">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>62</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Time Left</string>
     </property>
    </widget>
    <widget class="QLabel" name="time_left_number">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>25</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
